ItAcademyJava/Sprint1/Tasca04/Nivell1/Exercici1/src/s104n1e1/Gestion.java

```diff
@@ -1,39 +1,29 @@
 package s104n1e1;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class Gestion {
+	
 
-	public ArrayList<Month> monthListGen() {
-		ArrayList<Month> monthList = new ArrayList<Month>();
-
-		Month gener = new Month("gener");
-		Month febrer = new Month("febrer");
-		Month març = new Month("març");
-		Month abril = new Month("abril");
-		Month maig = new Month("maig");
-		Month juny = new Month("juny");
-		Month juliol = new Month("juliol");
-		Month agost = new Month("agost");
-		Month setembre = new Month("setembre");
-		Month octubre = new Month("octubre");
-		Month novembre = new Month("novembre");
-		Month desembre = new Month("desembre");
+	public List<String> monthListGen() {
+		List<String> monthList = new ArrayList<>();

```

Comment:
I must note that although i am saying this, i did not follow my own advice instead using a Enum to store all of the months. I also have to load said enum, which i did with a for of loop. I did it that way because i'm new to Java and I was exploring. It is not a standard way to do it.